assets/styl/_modules/choice.twig

```diff
@@ -0,0 +1,15 @@
+<form action="#">
+    <div class="choice">
+        <legend>Choississez une option</legend>
+        <ul class="choice-list g g-withgutters">
+            <li class="g-col-6">
+                <input type="radio" id="radio-1" name="test" value="option1">
+                <label for="radio-1">Option 1</label>
+            </li>
+            <li class="g-col-6">
+                <input type="radio" id="radio-2" name="test" value="option2">
+                <label for="radio-2">Option 2</label>
+            </li>
+        </ul>
+    </div>
+</form>
```

Comment:
Empty newline missing

Reply:
So... please someone help with this empty new line. 
I have an empty new line at the end of my file, but when I push, it's not visible on github. @ryuran told me it's not supposed to be visible. @mallier I added 2 empty lines in order to have 1 visible on github. ^^' that's no good. 
If the empty line is not supposed to be visible, that was OK in the beginning ^^'

Reply:
It's ok now

Reply:
You should configure your editor to use a plugin of http://editorconfig.org/#download
Colette use a file called `.editorconfig` which define these kind of rules.

Reply:
Right ^^